frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightTabs.tsx

```diff
@@ -19,11 +20,13 @@ enum PipelineRunNodeTabs {
 type PipelineRunDrawerRightTabsProps = {
   task: PipelineRunTaskDetails;
   taskReferences: TaskReferenceMap;
+  parameter?: PipelineRunTaskParam[];
```

Comment:
Likewise this prop should be called `parameters` and should probably not be optional.